### ISSUE TITLE
Product page scrolling automatically to the bottom of the page if Smooth Scrolling activated

### ISSUE THREAD

**Issue by sptaule** [soflyy/oxygen-bugs-and-features#1252]
**The bug**
If you have the smooth scrolling activated, you may encounter a weird behavior when Adding/Editing a WooCommerce product. When loaded, the page scrolls to bottom automatically.

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**To Reproduce**
1. Go to Builder Editor > Global styles > Scripts
2. Activate 'Smooth scrolling' option
3. Go to one of your WooCommerce product through backend
4. See auto scroll

**Expected behavior**
The page loads and doesn't scroll down automatically to the bottom

**Desktop (please complete the following information):**
- OS : Windows 10
- Browser : Firefox and Edge
 - Oxygen Version 3.3.1

**Comment by Rados51** [comment]
```
jQuery(".woocommerce-tabs li a").off("click");
jQuery(".woocommerce-tabs li a").click(function (e) {
e.preventDefault();
jQuery(".woocommerce-tabs li").removeClass("active");
jQuery(this).parent().addClass("active");
jQuery(".woocommerce-Tabs-panel:visible").hide();
jQuery(".woocommerce-Tabs-panel:visible").hide();
jQuery(jQuery(this).attr("href")).show();
return false;
});
```

https://gist.github.com/.../e3beaa8e2e3256cc4efa0f8527b627aa

**Comment by Spellhammer** [comment]
Fixed in 3.8.
